gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java

```diff
@@ -1015,65 +1020,101 @@ public void setSharePageMediaSites(List<Constants.SocialMediaSites> value) {
      * Returns whether the FeedBack link of the footer is to be displayed.
      * 
      * Set at application level via "goc.webtemplate.showfeedbacklink" property in cdn.properties, 
-     * can be overriden programatically.  
+     * can be overriden programatically.
+     * 
+     * @Depracted Starting with version v2.7.0, use get/setFeedbackLink instead

```

Comment:
I JUST noticed this, not sure if it made it in v4/master branch but, Deprecated is spelled wrong :)

Reply:
oh haha yea I had manually typed that in. Fixed!

(Should be good for v4)